smartsim/_core/control/controller.py

```diff
@@ -189,11 +234,26 @@ def stop_entity(self, entity: t.Union[SmartSimEntity, EntityList]) -> None:
                 )
                 self._jobs.move_to_completed(job)
 
-    def stop_entity_list(self, entity_list: EntityList) -> None:
+    def stop_db(self, db: Orchestrator) -> None:
```

Comment:
Is there any easy way we can add something like `def stop(self)` into the API for EntityList and SmartSimEntity so the controller can avoid having custom methods for each type. 

The result would be that stop_entity and shutdown_db use <entity>.stop and the controller no longer needs to know exact details of db implementation like `cmd = [redis_cli, "-h", host_ip, "-p", str(port), "shutdown"]`

Reply:
Probably better for follow-up task if deemed useful